src/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=Hello, world!, initial-scale=1.0">
+    <title>Document</title>
+  </head>
+  <body>
+  Hello, world!
```

Comment:
```suggestion
    Hello, world!
```

use right code style